lib/manager/components/NoteForm.js

```diff
@@ -63,31 +63,29 @@ export default class NoteForm extends Component<Props, State> {
     )
     return (
       <form>
-        <Col xs={12} sm={stacked ? 12 : 4} md={stacked ? 12 : 6}>
+        <Col md={stacked ? 12 : 6} sm={stacked ? 12 : 4} xs={12}>
           <h3>{this.messages('postComment')}</h3>
           <Media>
             <Left>
               <img
                 alt={profile.email}
                 height={64}
-                width={64}
-                src={profile.picture} />
+                src={profile.picture}
+                width={64} />
             </Left>
             <Body>
-              <Panel
-                className='comment-panel'
-              >
-                <Panel.Heading>
+              <Panel className='comment-panel'>
+                <Panel.Heading><Panel.Title componentClass='h3'>
                   <Heading>
```

Comment:
Do we need a duplication of these headings? The link element is rendering with the regular link color (previously, it rendered black)

Reply:
I've achieved parity with the dev branch by removing the new `Panel.Title` tags in 566bef2e3b25f7411ef8ad1d0183d5763c185b39